web/themes/joinup/prototype/sass/components/_button.sass

```diff
@@ -49,6 +49,9 @@
   z-index: 10
   position: relative
 
+.button__container
```

Comment:
Feels a bit hacky to be honest. I don't mind it. But if I were you, i'd make a simple mixin or helper to be able to re-use this rule for layouts and similar with a parameter of the spacing.

For example, there's already a quite similar scenario:
```
// Dependiencies
// --------------------------------------------------
.page__title-wrapper
  .details
    margin: 10px 0
    +breakpoint(mobile)
      margin: 0
```